### DISCUSSION TITLE
Default filters template

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#15189] [Feature improvement]
It would be very helpful, if we can define default set of filters for Roles/Profiles. Currently every new user need to struggle creating own filters. Creating default filters for all by admin, sometimes is not possible due to different roles configuration

**Comment by garryatbrighton** [comment]
It is already in YF /index.php?module=CustomView&parent=Settings&view=Index
However it returning errors as attached
![image](https://user-images.githubusercontent.com/80203589/133509744-b0afbfdb-fa03-4e9f-b71f-7744fb310c3e.png)

**Reply by rskrzypczak** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/issues/15190

**Comment by mariuszkrzaczkowski** [comment]
look at it, is it enough?
https://gitdeveloper.yetiforce.com/index.php?module=CustomView&parent=Settings&view=Index
![2021-09-16_07-24-44](https://user-images.githubusercontent.com/10198654/133554632-92c4efe8-3bcd-4fb9-b2a9-a70ff080b52b.png)
![2021-09-16_07-25-19](https://user-images.githubusercontent.com/10198654/133554635-d9e19235-08da-432e-874f-9633f4c340b1.png)
